finger/views.py

```diff
@@ -29,6 +30,54 @@ def upload_json(request):
         'form': form,
     })
 
+@login_required
+def mail_members(request):
+    if not (request.user.is_staff and request.user.is_superuser):
+        return HttpResponseNotAllowed()
+
+    if request.method == 'POST':
+        form = MailMembersForm(request.POST)
+    else:
+        form = MailMembersForm()
+
+    if form.is_valid():
+        members = form.get_members()
+
+        subject = form.cleaned_data['subject']
+        message = form.cleaned_data['message']
+        sender = request.user.username + '@stacken.kth.se'
+
+        recipients = form.cleaned_data['recipients']
+        if recipients:
+            recipients = [("", recipient) for recipient in recipients.split(',')]
+        else:
+            # todo send to all email addresses
+            recipients = [(f"{member.first_name} {member.last_name}", member.email) for member in members]
```

Comment:
`send.py` sent two emails, one to `{member.email}` and one to `{member.username}@stacken.kth.se`, the email is an optional external address that all users have not set.

Reply:
Also note, due the fact that we now import members without an account ... you probably also should check if their account is active (have an stacken-account) before we try to send emails to `{member.username}@stacken.kth.se`.

Reply:
Klart.